Azure-Samples/msdocs-flask-postgresql-sample-app
# Make code compatible with Alembic

## Purpose

Currently, this app does not quite run correctly, as it is mixing two ways of doing things: using Alembic for migrations versus using db.create_all(). It does end up successfully creating the tables due to db.create_all(), but then "flask db upgrade" actually accomplishes nothing.

So this PR fixes the situation by removing db.create_all() and moving some imports around to avoid a circular imports issue when running the upgrade command. This PR also adds the first migration file to the repo, which will be run when someone runs "flask db upgrade". That makes this repo more similar to the Django repo, which does have the initial migration file.

This PR also improves the ordering of the imports. I ran them through isort since I had to change them anyway.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[X] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

*  Get the code
* Make the .env file
* pip install -r requirements.txt
* flask db upgrade
* flask run


* Test the code

You should see the website and be able to add a restaurant.
